pyccel/ast/sympy_helper.py

```diff
@@ -25,7 +25,7 @@
 
 from .datatypes import NativeInteger
 
-from .variable  import Variable, PyccelArraySize
+from .variable  import Variable, NumpyArraySize
```

Comment:
The class `NumpyArraySize` is defined in module `pyccel.ast.internals` (although I would like to define it in module `pyccel.ast.numpyext`), and it should be imported from there directly.

Here instead it is imported from module `pyccel.ast.variable`, which also imports it. This is allowed in Python, but is definitely not good practice. This problem was not introduced by this PR, but it should be fixed now.